packages/lib/modules/pool/PoolList/PoolListFilters.tsx

```diff
@@ -64,7 +64,7 @@ export function useFilterTagsVisible() {
 
   useEffect(() => {
     setIsVisible(
-      networks.length > 0 || poolTypes.length > 0 || minTvl > 0 || poolCategories.length > 0
+      networks.length > 0 || poolTypes.length > 0 || minTvl > 0 || poolCategories.length > 0,
```

Comment:
This seems like an incorrect change?

Reply:
the default was changed in prettier v3: https://prettier.io/docs/en/options.html#trailing-commas

Reply:
Yeah that's fine but this particular usage looks weird to me. `aFunction(someParam,)` does not look right?

Reply:
Whatever part of the rule is causeing that, I think we should change. Trailing commas for arrays and objects are fine.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#trailing_commas_in_functions

Reply:
I'm not a fan, but if you want to keep it `all` instead of `es5` then let's do it. We can easily change later.

Reply:
added `trailingComma: "es5",` to prettier config